makerspace/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from django.contrib.auth.decorators import login_required
+from makerspace.views import ToolsView, ToolView, EditToolView, CreateToolView, \
+    AdminToolView, DeleteToolView
+
+
+urlpatterns = [
```

Comment:
How does one reach these urls? I se no link in the header?

Reply:
Editing or creating a tool can be accessed through the admin panel. Regarding the tool page, you cannot possibly reach it unless you have the correcet URL. Rather than linking to each tool which is what we are currently doing now, we could similarly refer to a overview page of all available tools instead:  `/makerspace/tools`.

Reply:
If I understand you correctly, @Maisergodt, should we add a link to `/makerspace/tools` on the `/makerspace` page? If that's the case, then it's fine by me 😊 What do you think, @hermabe?

Reply:
There needs to be a link the users/non-make-members can access the tools page. I didn't see a link. One could do it in the makerspace contentbox, but something more permanent may be better. I am not sure where it fits best. We could maybe add a dropdown in the header, but we haven't used that anywhere else so it breaks the pattern.

Reply:
Yes, I was thinking of adding a link in the makerspace contentbox page. Something like this such that regular users can access it as well. 
![image](https://user-images.githubusercontent.com/43437044/83678581-43616300-a5de-11ea-80aa-4f26ceda6ab8.png) 

I agree a permanent solution would be better. Although I am not certain where it's suitable.

Reply:
I'm not against a link in the contentbox. If we were to have a permanent link i think it should be either above or below the contentbox.

Reply:
The newest commits has added a dropdown menu with links to "makerverkstedet", "regler" and "verktøy". This is a solution to what was talked about here.